cmd/kube-controller-manager/controller-manager.go

```diff
@@ -26,8 +26,8 @@ import (
 	"os"
 	"time"
 
-	"k8s.io/apiserver/pkg/util/logs"
 	"k8s.io/kubernetes/cmd/kube-controller-manager/app"
+	"k8s.io/kubernetes/pkg/kubectl/util/logs"
```

Comment:
??

Reply:
yeah... I have no idea. I will fix this.